pytests/func/test_breakpoints.py

```diff
@@ -8,16 +8,16 @@
 import os.path
 import pytest
 import sys
-from pytests.helpers.timeline import Event
+from pytests.helpers.timeline import Event, Response
 from pytests.helpers.pathutils import get_test_root, compare_path
-from pytests.helpers.session import START_METHOD_LAUNCH, START_METHOD_CMDLINE
+from pytests.helpers.session import START_METHOD_LAUNCH, START_METHOD_CMDLINE, START_METHOD_IMPORT
 
 BP_TEST_ROOT = get_test_root('bp')
 
 
-@pytest.mark.parametrize('start_method', [START_METHOD_LAUNCH, START_METHOD_CMDLINE])
+@pytest.mark.parametrize('start_method', [START_METHOD_LAUNCH, START_METHOD_CMDLINE, START_METHOD_IMPORT])
```

Comment:
This is going to be the new default for all tests, right? (i.e. unless a test has a good reason to exclude some method, it should cover all of them). If so, let's make the `start_method` fixture return this by default, and drop `@parametrize` on all tests that use the default.

Reply:
Agreed. Will make the change.